src/ui/RelevanceInspector/RankingInfoParser.ts

```diff
@@ -167,7 +167,7 @@ const parseTermsWeights = (termsWeight: RegExpExecArray | null): IListOfTermsWei
 };
 
 const parseQREWeights = (value: string): IListOfQRE[] => {
-  const REGEX_EXTRACT_QRE_WEIGHTS = /(Expression:\s".*")\sScore:\s(?!0)([0-9]+)\n+/g;
+  const REGEX_EXTRACT_QRE_WEIGHTS = /(Expression:\s".*")\sScore:\s(?!0)([-0-9]+)\n+/g;
```

Comment:
This would allow negative symbols in the middle or end of the numbers.

```suggestion
  const REGEX_EXTRACT_QRE_WEIGHTS = /(Expression:\s".*")\sScore:\s-?(?!0)([0-9]+)\n+/g;
```